spec/acceptance/golden_masters_spec.rb

```diff
@@ -23,11 +23,11 @@
     let(:db) { Sequel.sqlite }
     let(:storage) { RetirementSwap::Storage::Database.new(db) }
     let(:output) { RetirementSwap::Output::IOWriter.new(StringIO.new) }
-    let(:retirement_swap) { RetirementSwap::SwapAlgorithm.new(storage, output) }
+    let(:processor) { RetirementSwap::Processor.new(storage, output, "52c1cf443ae7407d88000001" => {"algorithm" => "swap"}) }
```

Comment:
Line is too long. [125/80]<br>Space inside { missing.<br>Space inside } missing.